### PR TITLE
fix(Global Search/Tagging): regen services with latest API definition

### PR DESCRIPTION
**Pull request by francescadecicco**
## PR summary
defined tracing headers for GhoST search and tagging apis.
deprecated transaction-id  header parameter

Screenshot 2023-09-19 at 18 29 33

Screenshot 2023-09-19 at 18 28 35

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -14521,9512 +14521,5 @@
         "url": "https://github.com/sponsors/sindresorhus"
       }
     }
-  },
-  "dependencies": {
```
Comment (padamstx) [inline package-lock.json:14525]: I'm a little suspicious of this change :)
Why did you need to delete this entire "dependencies" field?
In general, when updating the project you should run `npm ci` in your sandbox to make sure npm downloads the precise versions of dependencies found in package-lock.json.
Here is a little more info about this:
https://github.com/IBM/platform-services-node-sdk/blob/main/update_service.md#initial-project-setup

**Review by padamstx — CHANGES_REQUESTED** [review]
Need to revert the change to package-lock.json.  Otherwise, the PR looks good.

File: package-lock.json (hunk 1 above)
Comment (francescadecicco) [reply package-lock.json]: thanks for suggestion, i reverted the changes

**Pull request merged**